consensus/istanbul/backend/announce.go

```diff
@@ -109,16 +109,67 @@ func (sb *Backend) sendAnnounceMsgs() {
 	sb.announceWg.Add(1)
 	defer sb.announceWg.Done()
 
-	ticker := time.NewTicker(time.Minute)
+	// Send out an announce message when this thread starts
+	go sb.sendIstAnnounce()
+
+	const (
+		// In this state, send out an announce message every 1 minute until the first peer is established
```

Comment:
Constants are generally still camelCase (e.g. https://golang.org/pkg/time/#pkg-constants), also it may be better to move these constants to the package level.
You might also make them a new type, e.g. `type AnnounceFrequencyState int`

Reply:
Feel free to skip this if we need to submit this PR. I understand we will rework this soon anyway